src/content/docs/en/reference/dev-overlay-plugin-reference.mdx

```diff
@@ -247,26 +247,87 @@ The dev overlay includes a set of web components that can be used to build plugi
 
 ### `astro-dev-overlay-window`
 
-Shows a window with optionally a title and an icon.
-
-`window-title` is a string that will be shown at the top of the overlay window. `window-icon` can either be a string of a SVG file or an icon from [the icon list](#icons).
+Shows a window.
 
 The slot of the component will be used as the content of the window.
 
 ```html
-<astro-dev-overlay-window window-title="MyWindows" window-icon="astro:logo">
+<astro-dev-overlay-window>
   <p>My content</p>
 </astro-dev-overlay-window>
 ```
 
+When building a window using JavaScript, remember that slotted content must go inside the light DOM of the component.
+
+```js
+const myWindow = document.createElement('astro-dev-overlay-window');
+const myContent = document.createElement('p');
+myContent.textContent = 'My content';
+
+// GOOD:
+myWindow.appendChild(myContent);
+
+// BAD:
+myWindow.shadowRoot.appendChild(myContent);
+```
+
+### `astro-dev-overlay-button`
+
+Shows a button.
+
+The slot of the component will be used as the content of the button.
+
+```js
+const myButton = document.createElement('astro-dev-overlay-button');
+myButton.textContent = 'Click me!';
+myButton.buttonStyle = "purple";
+myButton.size = "medium";
+
+myButton.addEventListener('click', () => {
+  console.log('Clicked!');
+});
+```
+
+#### `size`
+
+The size of the button, possible values are `small`, `medium` and `large`.
+
+#### `button-style`
+
+The style of the button, possible values are `ghost`, `outline`, `purple`, `gray`, and `red`. When using `ghost`, only the content of the button will be shown.
+
+In JavaScript, this property can be set using the `buttonStyle` property, as to avoid conflict with the native `style` property.
```

Comment:
```suggestion
In JavaScript, set this property using the `buttonStyle` property to avoid conflict with the native `style` property.
```
They might not *have* to do this, but it sounds like it's just gonna be safer if they do, so I'm fine with giving them a direct instruction to follow.